content/en/blog/falcosidekick-reponse-engine-part-7-cloud-functions.md

```diff
@@ -0,0 +1,366 @@
+---
+title: "Kubernetes Response Engine, Part 7: Falcosidekick + Cloud Functions"
+date: 2021-06-25
+author: Furkan Türkal
+slug: falcosidekick-reponse-engine-part-7-cloud-functions
+---
+
+> _This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`._
+>
+> See other posts:
+>
+> * [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> * [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
+> * [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
+> * [Kubernetes Response Engine, Part 4 : Falcosidekick + Tekton]({{< ref "/blog/falcosidekick-reponse-engine-part-4-tekton" >}})
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
```

Comment:
Please add part 6 in that list, and part 7 in list of all other posts

Reply:
Thanks, done!